department-of-veterans-affairs/vets-website
# VAGOV-5006: refactor health services to be data-driven from api

## Description 
- sort region health services by weight 
- sort region health services by parent weight
- group region health services by parent name
- no more hard coded taxonomy categories

## Testing done
locally with staging data

pre-req: no "orphaned" Regional Health Service nodes, i.e. a Regional Health Service has a parent Health Service Taxonomy as its `fieldServiceNameAndDescripti` - only children may go in this slot

to test:
1. re-build site
2. go to https://staging.va.gov/pittsburgh-health-care/health-services/
3. everything should look the same, except, the order should be exactly what it is in the CMS
4. you can check the order in the CMS here: http://staging.cms.va.gov/admin/structure/taxonomy/manage/health_care_service_taxonomy/overview
5. if there are "orphaned" services, delete them and re-build

## Screenshots
![localhost_3001_pittsburgh-health-care_health-services_ (6)](https://user-images.githubusercontent.com/19178435/62585362-492ad280-b86d-11e9-9aab-90428566644d.png)

## Acceptance criteria
- [ ] services are in the exact order as the CMS 

## Definition of done
